static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -333,23 +314,25 @@ function ReleaseContext(props: BaseContextProps) {
       </ReleaseContextContainer>
     );
 
-  const renderIssueCountAndAuthors = () =>
+  const renderReleaseDetails = () =>
     data && (
       <ReleaseContextContainer data-test-id="quick-context-release-issues-and-authors-container">
         <ContextRow>
           <div>
-            <ContextHeader>{t('New Issues')}</ContextHeader>
-            <ReleaseStatusBody>{data.newGroups}</ReleaseStatusBody>
+            <ContextHeader>{t('Created')}</ContextHeader>
+            <ReleaseBody>
+              <TimeSince date={data.dateCreated} />
+            </ReleaseBody>
           </div>
           <div>
-            <ReleaseAuthorsTitle>{getCommitAuthorTitle()}</ReleaseAuthorsTitle>
-            <ReleaseAuthorsBody>
-              {data.commitCount === 0 ? (
-                <IconNot color="gray500" size="md" />
-              ) : (
-                <AvatarList users={data.authors} />
-              )}
-            </ReleaseAuthorsBody>
+            <ContextHeader>{t('Last Event')}</ContextHeader>
```

Comment:
The PR description says that you removed "Last Event", just want to confirm that you may have meant "First Event"? I think that makes sense